README.md

```diff
@@ -60,10 +54,46 @@ programmer does not need to care and the compiler will do this work. As you can
 see, it is even possible to access the argument features of some feature from
 outside.
 
-The example also demonstrates pre- and postconditions in Fuzion. The result of
-the `get_name` feature should not be `Fuzion`, and the name given to `greet`
-should not be `World`. Pre- or postcondition failures will cause the program to
-panic.
+```
+ex_gcd is
+  max(a, b i32) i32 is
+    if a > b
+      a
+    else
+      b
+
+  common_divisors_of(a, b i32) list i32 is
+    x := max a.abs b.abs
+    y := 1..x
+    y.as_list.flatMap i32 (i->
+      if (a % i = 0) && (b % i = 0)
+        [-i, i].as_list
+      else
+        lists.empty i32)
+
+  gcd(a, b i32) i32
+    pre
+      safety: (a != 0 || b != 0)
+    post
+      safety: a % result = 0,
+      safety: b % result = 0,
+      pedantic: (common_divisors_of a b).reduce bool true (tmp,cur->tmp && (gcd.this.result % cur = 0))
+  is
+    if b = 0
+      a
+    else
+      gcd(b, a % b)
```

Comment:
could be written like: `gcd b a%b`

Reply:
should IMHO be
```
   gcd b (a % b)
```
or the variant @michaellilltokiwa suggested.